svo (refiller).xml

```diff
@@ -762,43 +471,36 @@ function svo.rf_refill(what)
     elseif what[i]:find("^(%w+)") then
       amount, potion = 1, what[i]:match("^(%w+)")
     end
-
+    
     if not concoctions[potion] and not toxins[potion] then
-      svo.echof("Don't know the ingredients for a '%s' potion :|", tostring(potion))
+      svo.echof("Don't know what a '%s' potion is :|", tostring(potion))
```

Comment:
Shouldn't assume what they tried was a potion, could be a toxin.  what a '%s' is :|, perhaps?